### PR TITLE
[FIX] dwg file format support

### LINKED ISSUE #11828: Could not upload Auto Cad drawing file on rocket
### Steps to reproduce:

1. Allow unlimit upload file type by clear the Comma-seperate list of media type then restart RocketChat server
2. On a direct message , click on ' +'  icon and select upload a Auto Cad drawing file  xxx.dwg from local computer
3. File upload and complete but not show the download icon 
4. Check it on the 'Files List' we could see the file xxxx.dwg but it grey out and could not download 
### Expected behavior:

The download link should show with a download icon, and show 

### Server Setup Information:

- Version of Rocket.Chat Server: 0.68.3
- Operating System: Ubuntu 18.04 - OS release  4.4.0-130-generic
- Deployment Method:  snap
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: v8.11.3
- MongoDB Version: 

### Relevant logs:

[34mI20180820-06:34:12.424(0) Exception while invoking method 'sendFileMessage' Error: Input buffer contains unsupported image format    => awaited here:     at Function.Promise.await (/snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:56:12)     at Promise.asyncApply (/snap/rocketchat-server/1317/programs/server/packages/rocketchat_file-upload.js:1745:49)     at /snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40  
[34mI20180820-06:40:05.268(0) Exception while invoking method 'sendFileMessage' Error: Input buffer contains unsupported image format    => awaited here:     at Function.Promise.await (/snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:56:12)     at Promise.asyncApply (/snap/rocketchat-server/1317/programs/server/packages/rocketchat_file-upload.js:1745:49)     at /snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40  
[34mI20180820-06:42:39.032(0) Exception while invoking method 'sendFileMessage' Error: Input buffer contains unsupported image format    => awaited here:     at Function.Promise.await (/snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:56:12)     at Promise.asyncApply (/snap/rocketchat-server/1317/programs/server/packages/rocketchat_file-upload.js:1745:49)     at /snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40  
[34mI20180820-07:21:42.679(0) Exception while invoking method 'sendFileMessage' Error: Input buffer contains unsupported image format    => awaited here:     at Function.Promise.await (/snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:56:12)     at Promise.asyncApply (/snap/rocketchat-server/1317/programs/server/packages/rocketchat_file-upload.js:1745:49)     at /snap/rocketchat-server/1317/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40  https://rocket.chat/docs/contributing/reporting-issues#gathering-logs -->

### PR DESCRIPTION
**Pull request by madguy02**
Closes #11828

### CODE REVIEW

**Comment by madguy02** [comment]
this PR adds a support for dwg files to be uploaded to Rocket.Chat. For some reason they are being treated as image files which causes the error `unsupported image format appears` .

**Comment by madguy02** [comment]
@sampaiodiego please review the changes once

File: packages/rocketchat-file-upload/server/methods/sendFileMessage.js [hunk 1]
```diff
@@ -32,6 +31,9 @@ Meteor.methods({
 			title_link_download: true,
 		};
 
+		// the condition to check fileType image/vnd.dwg is required as for
+		// some reason dwg format is considered as an image which throws error
+		if (file.type !== 'image/vnd.dwg') {
```
Comment (sampaiodiego) [inline packages/rocketchat-file-upload/server/methods/sendFileMessage.js]: please change this extra `if` to just another condition on the `if` above

**Comment by madguy02** [comment]
@sampaiodiego  removed the unnecessary `if` and added the logic within a single `if`

**Comment by sampaiodiego** [comment]
@madguy02 can you please fix the lint issues? thx

**Comment by tassoevan** [comment]
I've found that other `image/*` files are triggering the same issue. I'm closing this in favor of a more general solution in #12424.

**Pull request closed without merging**